examples/framework-lit/src/components/Counter.js

```diff
@@ -31,4 +29,4 @@ class Counter extends LitElement {
   }
 }
 
-customElements.define(tagName, Counter);
+customElements.define('my-counter', Counter);
```

Comment:
Our custom Lit Vite plugin will pick up on this and add an `export const __astroTagName = "my-counter"` export